packages/react-menu/src/MenuList.stories.tsx

```diff
@@ -297,7 +297,9 @@ export const RadioItemsControlled = () => {
 };
 
 export default {
-  // use the Components prefix to have the same URL as react-examples
-  title: 'Components/MenuList',
+  // use the Components prefix and (react-menu) suffix to have the same URL as react-examples
```

Comment:
After: #17884 the URLs no longer have the `react-*` suffix in the URL and I think @Hotell  kept that behaviour in his subsequent changes even in `react-examples`. At this point the suffix is purely naming convention

```suggestion
  // use the Components prefix and (react-menu) suffix to have the same naming convention as react-examples
```